hs/Css/Loading.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Css.Loading
+    (loadingStyles) where
+
+import Clay
+import Prelude hiding ((**))
+import Data.Monoid
+import Css.Constants
+
+{- loadingStyles
+ - Generates all CSS for the draw page. -}
+loadingStyles :: Css
+loadingStyles = do
+    spinnerCSS
+    smallCompassPosCSS
+    largeCompassPosCSS
+    spinCSS
+
+{- Properties of rotation animation. -}
```

Comment:
This style of documentation is different than what I usually use. I'm not certain how uniform the style is across the codebase, is there any reason why you decided to use `{- ... -}`?